docs/gateway/01_overview.md

```diff
@@ -0,0 +1,77 @@
+# LMS Gateway
+
+_See also:_
+
+ * [API Specification](02_api_spec.md) - Details of the end-points
+ * [Testing tools](03_testing_tools.md) - Testing tools and reference implementation
+
+## Overview
+
+The LMS Gateway allows you to access the main Hypothesis (`h`) API to retrieve 
+annotation and other data on behalf of and related to a certain user. 
+
+It will also provide you with contextual information about the current LMS 
+context in order to help you:
+
+ * Ask for the right things from `h`
+ * Map between LMS and `h` ids and concepts
+
+### The current implementation is a work in progress
+
+Details of the call structure, required fields and approach are likely to 
+change at this time without warning.
+
+### How it works
+
+You can authenticate with the Gateway end-point using a standard LTI basic 
+launch: an OAuth1 signed POST request. This will return to you a payload with:
+
+ * Instructions to call the `h` API and tokens to authenticate with it
+ * Contextual information about a course or assignment (users, groups, 
+ assignments etc.)
+
+You should then have everything you need to make any calls to the `h` API **as
+the user you passed to us**. This means you will see what they can see, and can
+act on behalf of that user.
+
+## Authentication
+
+Authentication happens in two steps:
+
+ * In the first step you will perform the signed OAuth1 request to the Gateway
+ * In the second step you will use tokens to authenticate with the `h` API
+
+### Getting started
+
+The Gateway is only available to certain partners. If you are interested in 
+using the Gateway, please contact [Hypothesis support](https://web.hypothes.is/help/)
+who can discuss your use case, and enable it for you.
```

Comment:
This isn't true yet, or maybe ever, but it's good to funnel people to support regardless.